### DISCUSSION TITLE
Discontinue support for JDK versions below 17[Feature]

### DISCUSSION THREAD

**Discussion by SuperEdison** [apache/dubbo#13998] [General]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dubbo/issues?q=is%3Aissue) and found no similar feature requirement.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Descriptions

can you guys make a standard about jdk version like spring, jdk must update to 17+ and not support below jdk17. I'm not questioning jdk8.

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
However, we still have a great number of users using JDK8. We can try it in Dubbo 4.x.

**Comment by SuperEdison** [comment]
yes, but how long you guys release dubbo 4.x, i am looking forward it
